src/components/enter-password/enter-password-controller.ts

```diff
@@ -128,6 +128,9 @@ export function enterPasswordPost(
           isPhoneNumberVerified: userLogin.data.phoneNumberVerified,
```

Comment:
Doesn't have to be done now but after we should tidy up and remove the `phoneNumberVerified` from the login response and just use `mfaMethodVerified`